### ISSUE TITLE
Translation: numbers in Spanish/Portuguese 

### ISSUE THREAD

**Issue by norbertschuler** [Plant-for-the-Planet-org/treecounter-app#2362]
**Describe the feature you'd like to see included in Plant-for-the-Planet App.**

The algorithm to show the treecounter number must be adapted for the newly added languages (“billion” vs. “trillion”).

![A8C24822-C8E4-47E6-B25A-BE6C19711D72](https://user-images.githubusercontent.com/1532418/82514508-2178d280-9b40-11ea-825a-129cd86e3982.jpeg)

**Comment by norbertschuler** [comment]
Resources in our translation system must be changed - the resource names are chosen by the short scale system rather than the long scale system (https://en.wikipedia.org/wiki/Long_and_short_scales):
```
        (isSingular
          ? i18n.t('label.thousand_singular')
          : i18n.t('label.thousand_plural')),
      ' ' +
        (isSingular
          ? i18n.t('label.million_singular')
          : i18n.t('label.million_plural')),
      ' ' +
        (isSingular
          ? i18n.t('label.billion_singular')
          : i18n.t('label.billion_plural')),
      ' ' +
        (isSingular
          ? i18n.t('label.trillion_singular')
          : i18n.t('label.trillion_plural')),
      ' ' +
        (isSingular
          ? i18n.t('label.quadrillion_singular')
          : i18n.t('label.quadrillion_plural')),
      ' ' +
        (isSingular
          ? i18n.t('label.quintillion_singular')
          : i18n.t('label.quintillion_plural'))

```

**Comment by norbertschuler** [comment]
@Chiagozie-Udeh I explained the resource names and the short and long scale system to the translator in the LingoHub chat and hope they will fix the wrong translations, e.g. in the Spanish language.

**Comment by Chiagozie-Udeh** [comment]
@norbertschuler I tried to fix them myself now, can you have a look and let me know if all okay

**Comment by norbertschuler** [comment]
@Chiagozie-Udeh "Billion" and "Trillion" seem to be right now if in Spain they expect to use that long scale system. The translation for "Quadrillion" and "Quintillion" in the short scale is not correct, it should be a million billion and a trillion in long scale), but as we never use them I don't mind:

<img width="1121" alt="Bildschirmfoto 2020-05-30 um 09 50 37" src="https://user-images.githubusercontent.com/1532418/83317973-11a57080-a25b-11ea-8bc4-d34292f71416.png">
<img width="1118" alt="Bildschirmfoto 2020-05-30 um 09 50 20" src="https://user-images.githubusercontent.com/1532418/83317976-15d18e00-a25b-11ea-862a-6b6b701d09d4.png">

**Comment by norbertschuler** [comment]
@Chiagozie-Udeh can you help decide if these numbers should be used on short or long scale for Portuguese (Brazil)? Currently they are translated like using the short scale as in English.

**Comment by Chiagozie-Udeh** [comment]
> @Chiagozie-Udeh "Billion" and "Trillion" seem to be right now if in Spain they expect to use that long scale system. The translation for "Quadrillion" and "Quintillion" in the short scale is not correct, it should be a million billion and a trillion in long scale), but as we never use them I don't mind:
> 
> <img alt="Bildschirmfoto 2020-05-30 um 09 50 37" width="1121" src="https://user-images.githubusercontent.com/1532418/83317973-11a57080-a25b-11ea-8bc4-d34292f71416.png">
> 
> <img alt="Bildschirmfoto 2020-05-30 um 09 50 20" width="1118" src="https://user-images.githubusercontent.com/1532418/83317976-15d18e00-a25b-11ea-862a-6b6b701d09d4.png">

I can still add the correct translation just to have all covered.

**Comment by Chiagozie-Udeh** [comment]
> @Chiagozie-Udeh can you help decide if these numbers should be used on short or long scale for Portuguese (Brazil)? Currently they are translated like using the short scale as in English.

I would think that we translate based on what we have in English, I am checking with the Spanish team too but that would seem right/. What do you think?

**Comment by norbertschuler** [comment]
According to Wikipedia they use the short scale system in Brazil and long scale in Europe: „Countries with usage of the long scale include most countries in continental Europe, and most that are French-speaking, Spanish-speaking[4] and Portuguese-speaking countries (except Brazil).
The short scale is used in most English-speaking and Arabic-speaking countries, in Brazil and several other countries.“
https://en.wikipedia.org/wiki/Long_and_short_scales

**Comment by norbertschuler** [comment]
> @norbertschuler I reached the Spanish team and here is the response:
> 
> "So to my knowledge we are using short scale in German (thousand billions = trillion).

No, we use the long scale system in German. We have own names for a million billion like milliard and billiard and trilliard:

Million: „Million“
Billion: „Milliarde“
Trillion: „Billion“
Quadrillion: „Billiarde“

> I would agree with this too unless you think strongly otherwise. Let me know. 

Every translator should know best what number names to use for his native language. Otherwise please study the Wikipedia pages.

**Comment by norbertschuler** [comment]
O.k., tomorrow I will export the language resources again and make the next release. Hopefully the number translations are finished.
